lib/spack/spack/compiler.py

```diff
@@ -274,11 +268,12 @@ def __init__(self, cspec, operating_system, target,
         def check(exe):
             if exe is None:
                 return None
-            _verify_executables(exe)
```

Comment:
This function doesn't do anything (now that `_verify_executables` has been removed).

Reply:
Good catch! I'll remove it.